addons/web/static/src/xml/base.xml

```diff
@@ -620,6 +620,7 @@
             title="env._t('Print')"
             items="printItems"
             icon="'fa fa-print'"
+            closeOnSelected="env.device.isMobile"
```

Comment:
:+1: for this line... but I'm not sure about the other ones in this commit... :trollface:

But could you add a test for this one, please? :angel:

Reply:
See also if it is possible to only replace this attribute in enterprise (using an XPath).